api/v1beta1/openstackmachine_types.go

```diff
@@ -164,9 +159,9 @@ type OpenStackMachineStatus struct {
 // +kubebuilder:resource:path=openstackmachines,scope=Namespaced,categories=cluster-api,shortName=osm
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=".metadata.labels.cluster\\.x-k8s\\.io/cluster-name",description="Cluster to which this OpenStackMachine belongs"
-// +kubebuilder:printcolumn:name="InstanceState",type="string",JSONPath=".status.instanceState",description="OpenStack instance state"
+// +kubebuilder:printcolumn:name="ServerID",type="string",JSONPath=".status.dependentResources.server.id",description="OpenStack instance ID"
```

Comment:
dependentResources should now be resources?

Reply:
Updated and re-tested on a running cluster:

```
> kc get openstackmachine
NAME             CLUSTER    SERVERID                               INSTANCESTATE   READY   MACHINE          AGE
test-foo-c4wdz   test-foo   a5500a2d-d1f8-4b8e-a7bb-2b94977eb893   BUILD                   test-foo-c4wdz   4m26s
```